docs/logic-apps/authentication.md

```diff
@@ -44,6 +44,24 @@ using (var logicApp = await LogicAppClient.CreateAsync(resourceGroup, logicAppNa
 }
 ```
 
+Or, alternatively you can choose to store the **client secret** in a secret store and retrieve it with an `ISecretProvider` instance.
+
+```csharp
+string clientSecretKey = "my-client-secret-key";
+ISecretProvider secretProvider = ...
+
+var authentication = LogicAppAuthentication.UsingServicePrincipal(tenantId, subscriptionId, clientId, clientSecretKey, secretProvider);
+```
+
+Both approaches allow you to pass along an optional Azure cloud environment to which the instance should authenticate and interact with the Logic Apps running on Azure.
```

Comment:
It's a bit hidden here, what if we move this to a dedicated section at the bottom, "Supporting all Azure clouds" for example, and list the auth options where this is supported?

If we add new ways in the future, it could then just be added there.

Reply:
Separated it below. Resolving.